Src/WindowsServer/WindowsServer.Shared/Implementation/AppServiceEnvironmentVariableMonitor.cs

```diff
@@ -27,7 +27,8 @@ internal sealed class AppServiceEnvironmentVariableMonitor : EnvironmentVariable
             "WEBSITE_SLOT_NAME",
             "WEBSITE_HOME_STAMPNAME",
             "WEBSITE_HOSTNAME",
-            "WEBSITE_OWNER_NAME"
+            "WEBSITE_OWNER_NAME",
+            "WEBSITE_RESOURCE_GROUP"
```

Comment:
Should we add comments to document where we expect these EnvVariables to be used?
Looking at this list, it would not be clear what applications or services support which variables.